nikola/plugins/misc/taxonomies_classifier.py

```diff
@@ -218,11 +218,15 @@ def _postprocess_path(self, path, lang, append_index='auto', dest_type='page', p
         `site.config['INDEX_FILE']` depending on `dest_type`, which can be
         `'feed'`, `'rss'` or `'page'`.
 
-        Finally, if `dest_type` is `'page'`, `page_info` can be `None` or a tuple
+        If `dest_type` is `'page'`, `page_info` can be `None` or a tuple
         of two integers: the page number and the number of pages. This will
         be used to append the correct page number by calling
         `utils.adjust_name_for_index_path_list` and
         `utils.get_displayed_page_number`.
+
+        If `alternative_path` is set to `True`, `utils.adjust_name_for_index_path_list`
```

Comment:
I added a very short explanation. (I want to avoid repeating the logic of `adjust_name_for_index_path_list` here.)